apollo-spaceport/README.md

```diff
@@ -0,0 +1,77 @@
+# Apollo Telemetry
+Transfer statistics from router to apollo ingress.
+
+## Authentication
+
+The server must be authenticated to submit messages to the Apollo ingress.
+Furthermore, the statistics must be submitted "to" a particular graph. In
+the existing products this is accomplished using [environment variables](https://www.apollographql.com/docs/federation/managed-federation/setup/#4-connect-the-gateway-to-studio).
+
+In the router, we have a configuration file which can be dynamically
+re-loaded, so it makes more sense to include this configuration here. There
+is a new optional section that looks like this:
```

Comment:
I originally intended to use environment variables, but they don't work well in the presence of "hot" reload. Since other aspects of configuration can be hot reload, making the graph reference and key not "hot" reload seemed worse than continuing to use environment variables.

Hard to decide which is "least worst" here. I can easily change this and use environment variables instead.

Reply:
They absolutely don't work well under the notion of hot-reload-ability, you're right. I think if someone really needs to hot-reload a config for the API key, then we could still let them use the config.  In that scenario, I'd say the config should override the env and probably emit a `warn` indicating that happened.

I've framed the (I think, 3) Product considerations above — let's just take this for an action item to discuss later.  What you've done works, and that's most important!

Reply:
(Generally aligned with @abernix here.)

I do think you should strongly consider using the exact terminology that Apollo Server uses for these configuration keys (which is something that we evolved over time and have been trying to standardize). That would be:

```
apollo:
  key:
  graphRef
```

While I recognize that it's usually better to use spelled-out names like "reference" instead of short-hand like "ref", I believe ref is the term that we more or less settled on for the platform (though there are places in the Studio UI that use reference).